pycqed/analysis_v2/timedomain_analysis.py

```diff
@@ -719,9 +748,16 @@ def get_cal_data_points(self):
                 cal_state = \
                     [k for k, idx in cal_states_rot_qb.items()
                      if idx == i][0]
-                self.cal_states_dict_for_rotation[qbn][cal_state] = \
-                    None if do_PCA and self.num_cal_points != 3 else \
-                        self.cal_states_dict[cal_state]
+                if do_PCA and self.num_cal_points != 3:
```

Comment:
What is special in the case of 3 cal points? Please add a comment.

Reply:
Before reading this, I had added the comment "Cannot do PCA for 3 cal states." Is this just not implemented or cannot be done conceptually?

Reply:
This was fixed in the meantime. Now it just checks whether do_PCA is True and the cal_states_dict for that qubit is empty